Tools-Override/resolveContract.targets

```diff
@@ -30,7 +31,9 @@
     <ItemGroup Condition="'@(ContractProject)' != ''">
       <!--Don't flow the values for Configuration --> 
       <ContractProject> 
-        <UndefineProperties>Configuration;%(ContractProject.UndefineProperties)</UndefineProperties> 
+        <UndefineProperties Condition="'$(HasBuildConfigurations)' != 'true' and '%(ContractProject.TargetGroup)' == ''">Configuration;%(ContractProject.UndefineProperties)</UndefineProperties>
```

Comment:
I pushed an update to https://github.com/dotnet/corefx/pull/14525 feel free to merge when green (I will if I'm online later).